go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/CPRT/roboclaw v0.0.0-20190825181223-76871438befc
 	github.com/Masterminds/sprig v2.22.0+incompatible
 	github.com/NYTimes/gziphandler v1.1.1
-	github.com/a8m/envsubst v1.3.0
+	github.com/a8m/envsubst v1.4.2
```

Comment:
all changes other than the removal of slam are the result of `go mod tidy` which is called automatically.

Eric said in the thread (see PR description) that we should update deps & that the circular dep was what was holding back deps being updated at a normal cadence.